pages/content/amp-dev/documentation/guides-and-tutorials/start/bento_guide.md

```diff
@@ -0,0 +1,303 @@
+
+[/tip]
+
+The example below triggers [`amp-accordion`'s toggle action](../../../documentation/components/reference/amp-accordion.md#toggle) when the user clicks the "Toggle Accordion" button.
+
+```html
+<amp-accordion id="my-accordion" disable-session-states>
+  <section>
+    <h2>Section 1</h2>
+    <p>Content in section 1.</p>
+  </section>
+  <section>
+    <h2>Section 2</h2>
+    <div>Content in section 2.</div>
+  </section>
+  <section>
+    <h2>Section 3</h2>
+    <img class="article-img" width="1024" height="682" src="https://raw.githubusercontent.com/ampproject/amp.dev/future/examples/static/samples/img/product2_1024x682.jpg">
+  </section>
+</amp-accordion>
+<button id="toggle">
+  Toggle Accordion
+</button>
+<script>
+  const toggleAccordion = async () => {
+    const accordion = document.querySelector('#my-accordion');
+    const api = await accordion.getApi();
+    api.toggle();
+  }
+  const toggleButton = document.querySelector('#toggle');
+  toggleButton.addEventListener('click', toggleAccordion);
+</script>
+```
+
+Read each [component's reference documentation](#available-bento-components) for a full list of available actions and events. 
+
+
+# Available Bento components  <a name="available-bento-components"></a>
+
+Bento supported AMP components are listed below:
+
+*   [amp-accordion](../../../documentation/components/reference/amp-accordion-v1.0.md)
+*   [amp-base-carousel](../../../documentation/components/reference/amp-base-carousel-v1.0.md)
+*   [amp-inline-gallery](../../../documentation/components/reference/amp-inline-gallery-v1.0.md)
+    *   amp-inline-gallery-pagination
+    *   amp-inline-gallery-thumbnails
+*   [amp-stream-gallery](../../../documentation/components/reference/amp-stream-gallery-v1.0.md)
+*   [amp-date-countdown](../../../documentation/components/reference/amp-date-countdown-v1.0.md)
+*   [amp-date-display](../../../documentation/components/reference/amp-date-display-v1.0.md)
+*   [amp-fit-text](../../../documentation/components/reference/amp-fit-text-v1.0.md)
+*   [amp-instagram](../../../documentation/components/reference/amp-instagram-v1.0.md)
+*   [amp-lightbox](../../../documentation/components/reference/amp-lightbox-v1.0.md)
+*   [amp-selector](../../../documentation/components/reference/amp-selector-v1.0.md)
+*   [amp-social-share](../../../documentation/components/reference/amp-social-share-v1.0.md)
+*   [amp-timeago](../../../documentation/components/reference/amp-timeago-v1.0.md)
+*   Video components
+    *   [amp-video](../../../documentation/components/reference/amp-video-v1.0.md)
+    *   [amp-youtube](../../../documentation/components/reference/amp-youtube-v1.0.md)
+
+# Bento in action
+
+The example below demonstrates how to include `amp-base-carousel` and `amp-accordion` in a non-AMP HTML page.
```

Comment:
it doesn't demo amp-accordion anymore.

Reply:
Have you tried embedding this as a live example?

Reply:
It's live now!